var/spack/repos/builtin/packages/py-elephant/package.py

```diff
@@ -14,8 +14,11 @@ class PyElephant(PythonPackage):
     git = "https://github.com/NeuralEnsemble/elephant.git"
 
     # list of GitHub accounts to notify when the package is updated.
-    maintainers("Moritz-Alexander-Kern")
+    maintainers = ["Moritz-Alexander-Kern"]
```

Comment:
Please restore the new maintainers directive (see https://spack.readthedocs.io/en/latest/packaging_guide.html#maintainers).

Reply:
Updated to directive for maintainers.